src/DGtal/dec/doc/modulePolygonalCalculus.dox

```diff
@@ -200,7 +200,31 @@ Operator  | Output | Description
 `calculus.M(f,lambda)`  | \f$ n_f\times n_f\f$ | Inner product operator on discrete 1-forms operator (lambda is a regularization parameter, see @cite degoes2020discrete)
 `calculus.LaplaceBeltrami(f,lambda)`|  \f$ n_f \times n_f\f$  | Laplace-Beltrami operator \f$ D^tMD \f$ (lambda is a regularization parameter, see @cite degoes2020discrete), the matrix is PSD
 
- 
+\subsection sectcovOPL Covariant Operators and Vector-Laplacian
+
+@image html images/poly/transport_to_face.png "transport of a vertex based vector field to face tangent plane"
+
```

Comment:
Explain briefly what is a covariant operator ?

Reply:
done